helm/harbor-operator-app/Chart.yaml

```diff
@@ -1,11 +1,20 @@
-apiVersion: v1
-appVersion: 0.5.1
-description: A Helm chart to deploy harbor-operator
-engine: gotpl
+apiVersion: v2
+name: harbor-operator
+description: Deploy Harbor Operator
+
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
```

Comment:
This version should align with our release version. Seems we didn't have a release yet, so this should be fine.